### PR TITLE
Fixed an issue with the PileupReweightingTool not working when runnin…

### PR DESCRIPTION
**Pull request by jmrolsson**
…g on condor

### CODE REVIEW

File: Root/BasicEventSelection.cxx [hunk 1]
```diff
@@ -32,7 +32,8 @@ BasicEventSelection :: BasicEventSelection (std::string className) :
     Algorithm(className),
     m_PU_default_channel(0),
     m_grl(nullptr),
-    m_pileup_tool_handle("CP::PileupReweightingTool/PileupToolName", nullptr),
+    // m_pileup_tool_handle("CP::PileupReweightingTool/PileupToolName", nullptr),
```
Comment (kratsg) [inline Root/BasicEventSelection.cxx]: remove this line

File: Root/BasicEventSelection.cxx [hunk 2]
```diff
@@ -32,7 +32,8 @@ BasicEventSelection :: BasicEventSelection (std::string className) :
...
+    m_pileup_tool_handle("CP::PileupReweightingTool/Pileup"), // fix to run on condor
```
Comment (kratsg) [inline Root/BasicEventSelection.cxx]: remove this comment

File: Root/BasicEventSelection.cxx [hunk 3]
```diff
@@ -517,7 +518,9 @@ EL::StatusCode BasicEventSelection :: initialize ()
 
     RETURN_CHECK("BasicEventSelection::initialize()", checkToolStore<CP::PileupReweightingTool>("Pileup"), "Failed to check whether tool already exists in asg::ToolStore" );
     //RETURN_CHECK("BasicEventSelection::initialize()", m_pileup_tool_handle.makeNew<CP::PileupReweightingTool>("CP::PileupReweightingTool/Pileup"), "Failed to create handle to CP::PileupReweightingTool");
-    RETURN_CHECK("BasicEventSelection::initialize()", m_pileup_tool_handle.make("CP::PileupReweightingTool/Pileup"), "Failed to create handle to CP::PileupReweightingTool");
+    // RETURN_CHECK("BasicEventSelection::initialize()", m_pileup_tool_handle.make("CP::PileupReweightingTool/Pileup"), "Failed to create handle to CP::PileupReweightingTool");
```
Comment (kratsg) [inline Root/BasicEventSelection.cxx]: remove this line

File: Root/BasicEventSelection.cxx [hunk 4]
```diff
@@ -517,7 +518,9 @@ EL::StatusCode BasicEventSelection :: initialize ()
...
+    RETURN_CHECK("initialize()", ASG_MAKE_ANA_TOOL(m_pileup_tool_handle, CP::PileupReweightingTool), "Could not make the tool"); // fix to run on condor
```
Comment (kratsg) [inline Root/BasicEventSelection.cxx]: remove the comment

File: Root/BasicEventSelection.cxx [hunk 5]
```diff
@@ -517,7 +518,9 @@ EL::StatusCode BasicEventSelection :: initialize ()
...
+
```
Comment (kratsg) [inline Root/BasicEventSelection.cxx]: remove this extra line

File: Root/BasicEventSelection.cxx [hunk 6]
```diff
@@ -526,7 +529,9 @@ EL::StatusCode BasicEventSelection :: initialize ()
     RETURN_CHECK("BasicEventSelection::initialize()", m_pileup_tool_handle.setProperty("DataScaleFactor", 1.0/1.16), "Failed to set pileup reweighting data scale factor");
     RETURN_CHECK("BasicEventSelection::initialize()", m_pileup_tool_handle.setProperty("DataScaleFactorUP", 1.0), "Failed to set pileup reweighting data scale factor up");
     RETURN_CHECK("BasicEventSelection::initialize()", m_pileup_tool_handle.setProperty("DataScaleFactorDOWN", 1.0/1.23), "Failed to set pileup reweighting data scale factor down");
-    RETURN_CHECK("BasicEventSelection::initialize()", m_pileup_tool_handle.initialize(), "Failed to properly initialize CP::PileupReweightingTool");
+    // RETURN_CHECK("BasicEventSelection::initialize()", m_pileup_tool_handle.initialize(), "Failed to properly initialize CP::PileupReweightingTool");
```
Comment (kratsg) [inline Root/BasicEventSelection.cxx]: remove this line

File: Root/BasicEventSelection.cxx [hunk 7]
```diff
@@ -526,7 +529,9 @@ EL::StatusCode BasicEventSelection :: initialize ()
...
+    RETURN_CHECK("BasicEventSelection::retrieve()", m_pileup_tool_handle.retrieve(), "Failed to properly retrieve CP::PileupReweightingTool"); // fix to run on condor
```
Comment (kratsg) [inline Root/BasicEventSelection.cxx]: remove the comment

File: Root/BasicEventSelection.cxx (hunk 2 above)
Comment (jmrolsson) [inline Root/BasicEventSelection.cxx]: ok

**Pull request merged**